world_universities_and_domains.json

```diff
@@ -130965,5 +130983,821 @@
+    "state-province": null,
+    "domains": [
+      "wou.edu"
+    ],
+    "country": "United States"
+  },
+  {
+    "web_pages": [
+      "https://rptu.de"
+    ],
+    "name": "Rheinland-Pfälzische Technische Universität Kaiserslautern-Landau",
+    "alpha_two_code": "DE",
+    "state-province": null,
+    "domains": [
+      "rptu.de"
+    ],
+    "country": "Germany"
+  },
+  {
+    "web_pages": [
+      "https://www.hs-nb.de/"
+    ],
+    "name": "Hochschule Neubrandenburg University of Applied Science",
+    "alpha_two_code": "DE",
+    "state-province": null,
+    "domains": [
+      "hs-nb.de"
+    ],
+    "country": "Germany"
+  },
+  {
+    "web_pages": [
+      "https://www.unir.net/"
+    ],
+    "name": "UNIR",
+    "alpha_two_code": "ES",
+    "state-province": null,
+    "domains": [
+      "comunidadunir.net"
+    ],
+    "country": "Spain"
+  },
+  {
+    "web_pages": [
+      "https://bws-hofheim.de/"
+    ],
+    "name": "Brühlwiesenschule Hofheim",
+    "alpha_two_code": "DE",
+    "state-province": null,
+    "domains": [
+      "bws-hofheim.de"
+    ],
+    "country": "Germany"
+  },
+  {
+    "web_pages": [
+      "https://www.eelu.edu.eg/"
+    ],
+    "name": "Egyptian E-Learning University",
+    "alpha_two_code": "EG",
+    "state-province": null,
+    "domains": [
+      "student.eelu.edu.eg"
+    ],
+    "country": "Egypt"
+  },
+  {
+    "web_pages": [
+      "https://www.itis.pr.it/"
+    ],
+    "name": "Itis Leonardo da Vinci",
+    "alpha_two_code": "IT",
+    "state-province": null,
+    "domains": [
+      "itis.pr.it"
+    ],
+    "country": "Italy"
+  },
+  {
+    "web_pages": [
+      "https://sfcollege.edu/"
+    ],
+    "name": "Santa Fe College",
+    "alpha_two_code": "US",
+    "state-province": null,
+    "domains": [
+      "sfcollege.edu"
+    ],
+    "country": "United States"
+  },
+  {
+    "web_pages": [
+      "https://www.pradita.ac.id/"
+    ],
+    "name": "Pradita University",
+    "alpha_two_code": "ID",
+    "state-province": null,
+    "domains": [
+      "pradita.ac.id"
+    ],
+    "country": "Indonesia"
+  },
+  {
+    "web_pages": [
+      "https://chdtu.edu.ua/"
+    ],
+    "name": "Cherkask State Technological University",
+    "alpha_two_code": "UA",
+    "state-province": null,
+    "domains": [
+      "chdtu.edu.ua"
+    ],
+    "country": "Ukraine"
+  },
+  {
+    "web_pages": [
+      "https://www.plmun.edu.ph/"
+    ],
+    "name": "Pamantasan Ng Lungsod Ng Muntinlupa",
+    "alpha_two_code": "PH",
+    "state-province": null,
+    "domains": [
+      "plmun.edu.ph"
+    ],
+    "country": "Philippines"
+  },
+  {
+    "web_pages": [
+      "https://u-paris.fr/"
+    ],
+    "name": "Université Paris Cité",
+    "alpha_two_code": "FR",
+    "state-province": null,
+    "domains": [
+      "etu.u-paris.fr"
+    ],
+    "country": "France"
+  },
+  {
+    "web_pages": [
+      "https://www.szpu.edu.cn/"
+    ],
+    "name": "Shenzhen polytechnic university",
```

Comment:
Shenzhen Polytechnic University seems to be the correct spelling

Reply:
True I'll change it